macos/FluentUI/Button/Button.swift

```diff
@@ -162,6 +195,12 @@ open class Button: NSButton {
 				return
 			}
 			updateContentTintColor()
+			if isPressed {
```

Comment:
You can also use the ternary operator here:

`buttonState = isPressed ? .selected : .normal`